src/Activities/ExtraActivityDimensions/ExtraActivityBaggageDimensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Omex.Extensions.Activities
+{
+	/// <inheritdoc/>
+	public class ExtraActivityBaggageDimensions : IExtraActivityBaggageDimensions
```

Comment:
This is excessively verbose, and kind of misleading.
They are not "extra", since the consuming service needs them.
They are just dimensions that are specific/custom to the use-case.
Suggest renaming to CustomBaggageDimensions or similar.